<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>きじが台地区住民自治協議会</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.ico</normaloff>icon.ico</iconset>
  </property>
  <widget class="QTableWidget" name="tbl_main">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>79</y>
     <width>1000</width>
     <height>668</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>1000</width>
     <height>668</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1000</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustToContents</enum>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="tabKeyNavigation">
    <bool>false</bool>
   </property>
   <property name="showDropIndicator" stdset="0">
    <bool>false</bool>
   </property>
   <property name="dragDropOverwriteMode">
    <bool>false</bool>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::ExtendedSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="gridStyle">
    <enum>Qt::SolidLine</enum>
   </property>
   <property name="sortingEnabled">
    <bool>false</bool>
   </property>
   <property name="cornerButtonEnabled">
    <bool>false</bool>
   </property>
   <property name="rowCount">
    <number>0</number>
   </property>
   <property name="columnCount">
    <number>0</number>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>50</number>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>419</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QComboBox" name="cmb_jitikai"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>自治会</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cmb_kumi"/>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>組</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cmb_han"/>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>班</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chk_del">
        <property name="text">
         <string>削除データを含む</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>氏名</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txt_name"/>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>住所</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txt_address"/>
      </item>
      <item>
       <widget class="QPushButton" name="btn_search">
        <property name="text">
         <string>検索</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btn_print">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>50</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>印刷</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>20</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>CSV読込</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_search</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>list_search()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>394</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>545</x>
     <y>40</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmb_jitikai</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>Form</receiver>
   <slot>jitikai_change()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>37</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>460</x>
     <y>6</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmb_kumi</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>Form</receiver>
   <slot>kumi_change()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>154</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>461</x>
     <y>23</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbl_main</sender>
   <signal>cellDoubleClicked(int,int)</signal>
   <receiver>Form</receiver>
   <slot>list_click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>671</x>
     <y>154</y>
    </hint>
    <hint type="destinationlabel">
     <x>707</x>
     <y>42</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_print</sender>
   <signal>clicked(bool)</signal>
   <receiver>Form</receiver>
   <slot>list_print()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>950</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>631</x>
     <y>28</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>csvload_click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>975</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>863</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>list_click()</slot>
  <slot>jitikai_change()</slot>
  <slot>kumi_change()</slot>
  <slot>list_search()</slot>
  <slot>list_print()</slot>
  <slot>csvload_click()</slot>
 </slots>
</ui>
